bot/resources/tags/return.md

```diff
@@ -1,35 +1,32 @@
 **Return Statement**
 
-When calling a function, you'll often want it to give you a value back. In order to do that, you must `return` it. The reason for this is because functions have their own scope. Any values defined within the function body are inaccessible outside of that function.
-
-*For more information about scope, see `!tags scope`*
+A value created inside a function can't be used outside of it unless you `return` it.
 
 Consider the following function:
 ```py
 def square(n):
-    return n*n
+    return n * n
 ```
-If we wanted to store 5 squared in a variable called `x`, we could do that like so:
+If we wanted to store 5 squared in a variable called `x`, we would do:
 `x = square(5)`. `x` would now equal `25`.
 
 **Common Mistakes**
 ```py
```

Comment:
Might help to enumerate the common mistakes here

Reply:
What change are you suggesting specifically?

Reply:
I'm suggesting to make the separation of the common mistakes clearer by saying sth like:
- Common mistake 1:
...
- Common mistake 2:
...

Reply:
I think the current way is good enough and more concise.